comp/core/agenttelemetry/impl/config.go

```diff
@@ -271,22 +202,36 @@ var defaultProfiles = `
           aggregate_tags:
             - transport
             - state
-        - name: dogstatsd.uds_origin_detection_error
-          aggregate_tags:
-            - transport
-        - name: dogstatsd.uds_connections
-          aggregate_tags:
-            - transport
+        - name: point.sent
+        - name: point.dropped
     schedule:
       start_after: 30
       iterations: 0
       period: 900
-  - name: core-status
-    status:
-      template: basic
-      extra:
-        runner.workers.count: '.runnerStats.Workers.Count'
-        runner.workers.utilization: '[.runnerStats.Workers | .. | objects | .Utilization] | add'
+  - name: database
+    metric:
+      exclude:
+        zero_metric: true
+      metrics:
+        - name: oracle.activity_samples_count
+        - name: oracle.activity_latency
+        - name: oracle.statement_metrics
+        - name: oracle.statement_plan_errors
```

Comment:
cc @sethsamuel (in upcoming in the 7.60, I have moved metrics to more distinct sections (aka profiles). This one is for DB